base/server/python/pki/server/deployment/scriptlets/security_databases.py

```diff
@@ -363,12 +363,15 @@ def destroy(self, deployer):
         if len(deployer.instance.tomcat_instance_subsystems()) > 0:
             return
 
+        logger.info('Removing %s', deployer.mdict['pki_client_dir'])
         if deployer.directory.exists(deployer.mdict['pki_client_dir']):
-            logger.info('Removing %s', deployer.mdict['pki_client_dir'])
-            pki.util.rmtree(deployer.mdict['pki_client_dir'])
+            pki.util.rmtree(deployer.mdict['pki_client_dir'],
```

Comment:
I think the `logger.info()` should be kept inside the conditional as in the original code since the client dir may not exist.